packages/schematics/migrations/update-6-1-1/files/karma.conf.ts__tmpl__

```diff
@@ -0,0 +1,33 @@
+// Karma configuration file, see link for more information
+// https://karma-runner.github.io/1.0/config/configuration-file.html
+
+import { join } from 'path';
+
+export const getBaseKarmaConfig = config => {
```

Comment:
Do not pass config to getBaseKarmaConfig. It complicates the interaction.

Just remove logLevel from here and set it in the project tsconfig.

```
const baseConfig = getBaseKarmaConfig(config);
  config.set({
    ...baseConfig,
    logLevel: config.LOG_INFO
    coverageIstanbulReporter: {
      ...baseConfig.coverageIstanbulReporter,
      dir: join(__dirname, '../../coverage/apps/projectName')
    }
  });
```